deployments/charts/kuma/README.md

```diff
@@ -69,6 +69,8 @@ A Helm chart for the Kuma Control Plane
 | ingress.annotations | object | `{}` | Additional deployment annotation |
 | ingress.nodeSelector | object | `{"kubernetes.io/arch":"amd64","kubernetes.io/os":"linux"}` | Node Selector for the Ingress pods |
 | kumactl.image.repository | string | `"kumactl"` | The kumactl image repository |
+| kubectl.image.registry | string | `"bitnami"` | The kubectl image registry |
+| kubectl.image.repository | string | `"kubectl"` | The kubectl image repository |
```

Comment:
shouldn't tag be also documented? And that by default it is empty value and it means is inferred from the cluster version